unfoldingWord/translationCore
# Bugfix #3486/wA Tool crashes on missing alignment data for verse.

#### This pull request addresses:

Issue: unfoldingWord-dev/translationCore#3486


#### How to test this pull request:

Needs to be tested with: https://github.com/translationCoreApps/word-alignment-tool/pull/14

Import usfm file contained in issue.  Then open word alignment tool to 1:81 - should not see crash.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3492)
